release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1272,6 +1272,43 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
+
+* Previously, a node port failed to expose properly because the `ipFamilyPolicy` field was set to `SingleStack` for the `kube-apiserver` resource. With this update, if the `ipFamilyPolicy` is set to `PreferredDualStack`, node port is exposed properly. (link:https://issues.redhat.com/browse/OCPBUGS-23350[*OCPBUGS-23350*])
+
+* Previously, after configuring the Open Virtual Network (OVN) for a hosted cluster, the `cloud-network-config-controller`, `multus-admission-controller`, and`ovnkube-control-plane` resources were missing the `hypershift.openshift.io/hosted-control-plane:{hostedcluster resource namespace}-{cluster-name}` label. With this update, after configuring the Open Virtual Network (OVN) for a hosted cluster, the `cloud-network-config-controller`, `multus-admission-controller`, and`ovnkube-control-plane` resources contain the `hypershift.openshift.io/hosted-control-plane:{hostedcluster resource namespace}-{cluster-name}` label. (link:https://issues.redhat.com/browse/OCPBUGS-19370[*OCPBUGS-19370*])
+
+* Previously, after creating a hosted cluster, to create a config map, if you used a name other than `user-ca-bundle`, the deployment if the Control Plane Operator (CPO) failed. With this update, you can use unique names to create a config map. The CPO is deployed successfully. (link:https://issues.redhat.com/browse/OCPBUGS-19419[*OCPBUGS-19419*])
+
+* Previously, hosted clusters with `.status.controlPlaneEndpoint.port: 443` would mistakenly expose the port 6443 for public and private routers. With this update, hosted clusters with `.status.controlPlaneEndpoint.port: 443` only expose the port 443. (link:https://issues.redhat.com/browse/OCPBUGS-20161[*OCPBUGS-20161*])
+
+* Previously, if the Kube API server is exposed by using IPv4 and IPv6, and the IP address is set in the `HostedCluster` resource, the IPv6 environment did not work properly. With this update, when the Kube API server is exposed by using IPv4 and IPv6, the IPv6 environment works properly. (link:https://issues.redhat.com/browse/OCPBUGS-20246[*OCPBUGS-20246*])
+
+* Previously, if the console Operator and Ingress pods were located on the same node, the console Operator would fail and mark the console cluster Operator as unavailable. With this release, if the console Operator and Ingress pods are located on the same node, the console operator no longer fails. (link:https://issues.redhat.com/browse/OCPBUGS-23300[*OCPBUGS-23300*])
```

Comment:
```suggestion
* Previously, if the console Operator and Ingress pods were located on the same node, the console Operator would fail and mark the console cluster Operator as unavailable. With this release, if the console Operator and Ingress pods are located on the same node, the console Operator no longer fails. (link:https://issues.redhat.com/browse/OCPBUGS-23300[*OCPBUGS-23300*])
```